interface-definitions/include/nat-rule.xml.i

```diff
@@ -18,6 +18,7 @@
         <help>NAT destination parameters</help>
       </properties>
       <children>
+        #include <include/firewall/fqdn.xml.i>
```

Comment:
FQDN should be `multi`, isn't it?
```
vyos@r14# set nat source rule 100 destination fqdn example.com
[edit]
vyos@r14# set nat source rule 100 destination fqdn n1.example.com
[edit]
vyos@r14# set nat source rule 100 destination fqdn n2.example.com
[edit]
vyos@r14# set nat source rule 100 destination fqdn n3.example.com
[edit]
vyos@r14# show nat source rule 100 destination 
+fqdn n3.example.com
[edit]
vyos@r14# 

```
one FQDN per rule is inconvenient

Reply:
We should steer users to use a group for multiple per rule.

Reply:
I agree, this is analogous to the firewall syntax, which supports either a group (including a domain group) or a single FQDN. Domain groups are supported in NAT.